pkg/commands/process/settings/policies/CR-001/rule.yml

```diff
@@ -0,0 +1,9 @@
+description: "Leaking sensitive data to loggers is a common cause of data leaks and can lead to data breaches. This policy looks for instances of sensitive data sent to loggers."
```

Comment:
@elsapet I wonder if `description` is still needed in `rule.yml`? I think currently it's only used in the docs, but the readme could take care of that.